noah_core/views.py

```diff
@@ -20,11 +21,11 @@ def post(self, request):
 class DonationView(View):
     def validate_input(self, body):
         ip = json.loads(body)
+        telpat = re.compile('^0?\d{10}$')
```

Comment:
phone number validation rules should ideally be the responsibility of a `Phonenumber` class. Doing that would enable the service to use google `phonenumber` lib later for better number validation